R/labeller.r

```diff
@@ -433,7 +435,10 @@ labeller <- function(..., .rows = NULL, .cols = NULL,
   if (!is.null(keep.as.numeric)) {
     .Deprecated(old = "keep.as.numeric")
   }
+
   dots <- list(...)
+
+
```

Comment:
Can I get you to revert these superfluous changed

Reply:
My apologies. Fixed in https://github.com/tidyverse/ggplot2/pull/4188/commits/6d488b6716c5228bc68f9fc1841888abbffa1565.

Reply:
No apology needed - thanks for your work on this